### DISCUSSION TITLE
502 bad gateway || returned when self host docker file

### DISCUSSION THREAD

**Discussion by yalmashagbah** [Budibase/budibase#12545] [Help]
trying to deploy budibase using docker container  but it keep returning 502 
*without the use of the docker-compose file
the docker container file :
```
FROM budibase/budibase:latest
EXPOSE 80
```
and the used run command is
 ```
docker run -d -t \
  --name=budibase \
  -p 10000:80 \
  contianer-name 
```

**Comment by melohagan** [comment]
Hey @yalmashagbah 

Looking at the docs page, it looks like you're missing the volume: https://docs.budibase.com/docs/docker

**Reply by yalmashagbah** [reply]
@melohagan 
adding the volume flag didn't solve the issue

**Reply by melohagan** [reply]
Did you get any errors when running the command? Are there any errors in logs?

**Reply by yalmashagbah** [reply]
@melohagan  no error just it return 502. locally it is working fine but i am trying to run it on the aws server

**Reply by melohagan** [reply]
ah well that guide is for local hosting. You may need to replace the `/local/path/data` volume path with something else. This might be relevant: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-volumes.html
